scaletest/terraform/action/prometheus.tf

```diff
@@ -0,0 +1,152 @@
+locals {
+  prometheus_helm_repo                  = "oci://registry-1.docker.io/bitnamicharts"
```

Comment:
I've used the Bitnami charts in the past but I've heard of some concerns with using them. cc @dannykopping

Reply:
Agreed; best to use the official one in my view.

Reply:
Will switch to that chart and request review when I've got that working 👍